inventory/templates/inventory/catalogue_for_type.html

```diff
@@ -37,33 +38,59 @@ <h1>Item catalogue for {{ item_type }}</h1>
                 <td>
                     {% if item.image %}
                         <a class="enlargable_image">
-                            <img style="height: 2em;" src={{ item.image.url }}>
+                            <img style="width: 2em;" src={{ item.image.url }}>
                         </a>
                     {% endif %}
-                    {{ item.name }}
+                </td>
+                <td style="width: 20%;">
+                    <div class="" data-toggle="tooltip" data-placement="bottom" title="{{ item.description }}">
+                        {{ item.name }}
+                    </div>
 
                 </td>
-                {% if can_change_items %}
-                    <td>
-                        <a href="{% url "inventory:catalogue_update_item" type_id=item_type.id item_id=item.id %}"
+                <td>
+                    {% if can_change_items %}
+                        <a href="{% url "inventory:catalogue_update_item" type_id=item_type item_id=item.id %}"
                            class="btn btn-sm btn-primary">
                             Edit
                         </a>
-                    </td>
-                {% endif %}
-                <td class="w-50">
-                    {{ item.description }}
+                    {% endif %}
+                </td>
+                <td>
+                    {% if can_maintain_ownerships %}
```

Comment:
Right now there are (at most) four buttons on a catalogue overview page, which is pretty crowded (edit item, edit ownership, add to committee, add to member). Furthermore, each of these buttons are also reachable from an item's link overview page.

I doubt it is needed to have shortcuts on such an overview page for stuff that won't be used daily. Hence, I suggest we replace all four of these buttons by a single "Edit" one that redirects to the link page, from where all the buttons can be accessed.

Reply:
I agree that eventually we need to lessen it, but I want it availlable while we still migrate the massive catalogue of items. It saves a lot of clicks in the initial inventory of boardgame and roleplaying items